q2_feature_classifier/_taxonomic_classifier.py

```diff
@@ -95,7 +95,7 @@ def _2(data: Pipeline) -> TaxonomicClassiferTemporaryPickleDirFmt:
 
 
 @plugin.register_transformer
-def _3(dirfmt: TaxonomicClassifierDirFmt) -> dict:
+def _3(dirfmt: TaxonomicClassifierDirFmt) -> Pipeline:
```

Comment:
QIIME 2 has to search for the transformer before it calls it, and `Pipeline` is part of the key it uses to identify this. Since the method wants a `Pipeline` it needs to find a transformer turning the artifact's view into that.